src/api/config/index.ts

```diff
@@ -25,6 +25,13 @@ export interface PackageTagConfig {
   };
 }
 
+export interface TagGroupConfig {
```

Comment:
Can we add a small comment in the source code that links/references the original interface defined in our backend repo? May help future readers